pkg/options/driver.go

```diff
@@ -26,7 +27,9 @@ type DriverTypes struct {
 
 // NewDriverTypes returns a new Enum configured for the driver types.
 func NewDriverTypes() *DriverTypes {
+	types := drivertype.GetTypes()
+	sort.Strings(types)
 	return &DriverTypes{
-		Enum: NewEnum(drivertype.GetTypes(), drivertype.TypeKmod),
+		Enum: NewEnum(types, drivertype.TypeKmod),
```

Comment:
To get a stable CLI help message, we need to sort the strings so that they always appear with same sorting.